ibex_core_tracing.core

```diff
@@ -33,26 +33,30 @@ parameters:
     paramtype: vlogdefine
 
   RV32E:
-    datatype: bool
+    datatype: int
     paramtype: vlogparam
 
   RV32M:
-    datatype: bool
+    datatype: int
```

Comment:
Any reason for that change?

Reply:
It's used inconsistently across the cores and I found with 'bool' I couldn't get it to work, end up with errors of the form

```
fusesoc run lowrisc_ibex_ibex_core_tracing_0.1: error: argument --RV32E: ignored explicit argument 'True'
```

`--RV32E=1` `--RV32E=True` and `--RV32E=true` all give the same result.

Potentially a fusesoc issue that I should file!

Reply:
I noticed that also. It wants `--RV32E`` only to set it to true (no argument).

Reply:
Aha, that may well be preferable, though I should also alter all of the other .core files to be consistent. Think I'll do that.

Reply:
Please file a fusesoc bug/feature request to allow the forms you tried (I thought it worked like that in the past, but I'm not sure). It certainly makes sense.